app/governance/components/votingChart/voteGraph.tsx

```diff
@@ -434,9 +401,11 @@ export const VoteGraphBox = ({
             }}
           />
         </div>
-        <Text font="rm_mono" className={styles.tableData} size="x-sm">
-          {"ACTIVE"}
-        </Text>
+        <Container className={styles.active}>
+          <Text font="proto_mono" className={styles.tableData} size="x-sm">
+            {"ACTIVE"}
```

Comment:
Wrapped the "Active" text inside a container to adjust the padding, margins and make it consistent with other table rows.